edits and updates (#10)
EpistasisLab/pmlb-manuscript: content/03.methods.md

The R library also includes a number of detailed "vignette" documents to help new users learn how to use the software.

PMLB now includes original data rows with missing data (i.e., NA). 
The core function of PMLB, `fetch_data()`, retains previous behavior by default, which excludes all rows with missing data (`dropna=True`). 
However, if the user chooses to treat the missing values differently, they can use `fetch_data()` with the option `dropna=False` to obtain the original dataset and apply their own removal or imputation method.

The new version of PMLB also allows the user to select datasets most similar to one of their own using the `nearest_datasets()` function. 

> Here, similarity between datasets is configurable to any number of metadata characteristics defined for the datasets. 

Reviewer: ```suggestion
Here, the similarity between datasets is configurable to any number of metadata characteristics.
```

> By default, the nearest datasets in terms of number of samples and features is returned. 

Reviewer: ```suggestion
By default, the names of the nearest datasets are returned based on the number of samples and features.
```

> This functionality should be helpful for users who would like to find PMLB datasets with similar characteristics to their own in order to test or optimize methods (e.g. hyperparameter tuning) for their desired problem without the risk of over-fitting to their dataset. 

Reviewer: ```suggestion
This functionality is helpful for users who look for PMLB datasets with similar characteristics to their own in order to test or optimize methods (e.g. hyperparameter tuning) for their desired problem without the risk of over-fitting to their dataset. 
```

Author: how about "wish to find"

Reviewer: Sounds great! Please feel free to merge when you're ready!
